README_en.md

```diff
@@ -0,0 +1,161 @@
+# soyclustering: Python clustering algorithm library for document clustering
+
+This package is implementation of **Improving spherical k-means for document clustering: Fast initialization, sparse centroid projection, and efficient cluster labeling** (Kim et al., 2020).
+
+Cosine is more appropriate than Euclidean to measure the distance between two documents which are represented with high-dimensional (or sometimes, sparse) vectors.
+However, scikit-learn k-means package only provides Euclidean based k-means not yet.
+In addition, labeling clusters is very helpful to interpret the clustering results.
+
+Spherical k-means works well both with sparse vector representation such as Bag-of-Words model or distributed representation such as Doc2Vec or other document embedding methods.
+In lower dimensional vector space, Silhouette score method is useful to define the number of clusters (`k`).
+However Silhouette score method does not works well anymore in high-dimensional vector space such as Bag-of-Words Models and Doc2Vec.
+One of heuristic methods to define the cluster numbers is to train k-means with large `k` first and then merge similar ones.
+The heuristic method is also useful to prevent Uniform effect which is the phenomenon that the size of all clusters leads similar.
```

Comment:
~ in preventing the uniform effect, which denotes that the size of all clusters will be similar